src/test/rustdoc/auxiliary/cross-crate-hidden-assoc-trait-items.rs

```diff
@@ -0,0 +1,15 @@
+#![crate_name = "cross_crate_hidden_assoc_trait_items"]
```

Comment:
Oops, that's left over from experimenting with stuff. I'm gonna remove it after the CI passes.